assets/layers/toilet_at_amenity/toilet_at_amenity.json

```diff
@@ -0,0 +1,475 @@
+        "nl": "Hoe breed is de deur van de rolstoeltoegankelijke toilet?",
+        "fr": "Quelle est la largeur de la porte des toilettes accessibles aux fauteuils roulants ?",
+        "de": "Wie breit ist die Tür zur rollstuhlgerechten Toilette?",
+        "da": "Hvad er bredden på døren til det kørestolsvenlige toilet?"
+      },
+      "render": {
+        "en": "The door to the wheelchair-accessible toilet is {canonical(toilets:door:width)} wide",
+        "nl": "De deur naar de rolstoeltoegankelijke toilet is {canonical(toilets:door:width)} wide",
+        "fr": "La porte des toilettes accessibles aux fauteuils roulants a une large de {canonical(toilets:door:width)}",
+        "de": "Die Tür zur rollstuhlgerechten Toilette ist {canonical(toilets:door:width)} breit",
+        "da": "Døren til det kørestolsvenlige toilet er {canonical(toilets:door:width)} bred"
+      },
+      "freeform": {
+        "key": "toilets:door:width",
+        "type": "pfloat"
+      }
+    },
+    "toilet.toilets-type",
+    "toilet.toilets-changing-table",
+    "toilet.toilet-changing_table:location",
+    "toilet.toilet-handwashing",
+    "toilet.toilet-has-paper",
+    {
+      "builtin": "description",
+      "override": {
+        "render": "{toilets:description}",
+        "freeform": {
+          "key": "toilets:description",
+          "type": "string"
+        }
+      }
+    }
+  ],
+  "filter": [
+    {
+      "id": "wheelchair",
+      "options": [
+        {
+          "question": {
+            "en": "Wheelchair accessible",
+            "nl": "Rolstoel toegankelijk",
+            "de": "Rollstuhlgerecht",
+            "es": "Accesible con sillas de ruedas",
+            "fr": "Accessible aux fauteuils roulants",
+            "da": "Tilgængelig for kørestolsbrugere"
+          },
+          "osmTags": {
+            "or": [
+              "toilets:wheelchair=yes",
+              "toilets:wheelchair=designated"
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "id": "changing_table",
+      "options": [
+        {
+          "question": {
+            "en": "Has a changing table",
+            "nl": "Heeft een luiertafel",
+            "de": "Mit Wickeltisch",
+            "es": "Tiene un cambiador",
+            "fr": "A une table à langer",
+            "da": "Har et puslebord"
+          },
+          "osmTags": "changing_table=yes"
+        }
+      ]
+    },
+    {
+      "id": "free",
+      "options": [
+        {
+          "question": {
+            "en": "Free to use",
+            "nl": "Gratis toegankelijk",
+            "de": "Nutzung kostenlos",
+            "fr": "Utilisation gratuite",
+            "da": "Gratis at bruge"
+          },
+          "osmTags": {
+            "or": [
+              "toilets:fee=no",
+              "toilets:fee=0",
+              "toilets:charge=0"
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "id": "is_open",
```

Comment:
There is a builtin filter for this, namely [here](https://github.com/pietervdvn/MapComplete/blob/fb277c693f4631edbd06d0198320f2bd23d05b41/assets/layers/filters/filters.json#L10)

Reply:
Changed, copied over from `toilet.json`, so changed there as well.